tidalapi/src/main/kotlin/com/tidal/sdk/tidalapi/generated/models/AlbumsMultiDataDocument.kt

```diff
@@ -21,22 +21,20 @@ import kotlinx.serialization.Serializable
 /**
  *
  *
- * @param `data` array of primary resource data
+ * @param `data`
  * @param links
  * @param included
  */
 
 @Serializable
 data class AlbumsMultiDataDocument(
 
-    // array of primary resource data
-
     @SerialName(value = "data")
     val `data`: kotlin.collections.List<AlbumsResource>? = null,
 
     @SerialName(value = "links")
     val links: Links? = null,
 
     @SerialName(value = "included")
-    val included: kotlin.collections.List<AlbumsMultiDataDocumentIncludedInner>? = null,
+    val included: kotlin.collections.List<AlbumsSingleDataDocumentIncludedInner>? = null,
```

Comment:
this looks incorrect. Probably a generator issue

Reply:
@ziad-halabi9  @usmanrafi I have spent some time researching this issue, until it dawned on me:
This is correct. The single and multi data documents have the same `included` definition, so they can use the same inner type. The `single` and `multi` disparity is a little unfortunate and quite misleading, but I'd suggest to accept it for now. So, despite the odd look I believe the generated code is correct.